src/workers/personalizedDigestEmail.ts

```diff
@@ -210,10 +216,36 @@ const worker: Worker = {
         groupId: 23809,
       },
       category: 'Digests',
+      batchId: emailBatchId,
       ...variationProps,
     };
 
+    const personalizedDigestWithLastSendDate = await con
+      .getRepository(UserPersonalizedDigest)
+      .findOne({
+        select: ['lastSendDate'],
+        where: {
+          userId: personalizedDigest.userId,
+        },
+      });
+
+    if (
+      personalizedDigestWithLastSendDate?.lastSendDate &&
+      isSameWeek(currentDate, personalizedDigestWithLastSendDate.lastSendDate)
```

Comment:
Can we use `isSameDay` instead? Pretty sure we would need to support daily digests for some users and this will be easily missed